lighthouse-core/gather/gather-runner.js

```diff
@@ -91,9 +92,17 @@ class GatherRunner {
       }).then(_ => {
         // Clears storage for origin.
         return driver.clearDataForOrigin(options.url);
+      }).then(_ => {
+        // Check Service Workers running
+        return driver.checkForMultipleTabsAttached(options.url);
```

Comment:
I feel like `setupDriver` has a weird order. Feel free to punt to a followup PR, but I feel like the multiple tabs check should come before `clearingDataForOrigin`, for instance (why bother if the SW clear isn't going to work anyways?). Not sure of optimal order, though.